src/streams/StreamIcon.js

```diff
@@ -18,8 +18,9 @@ export default function StreamIcon(props: Props): Node {
   const { color, style, isPrivate, isMuted, isWebPublic, size } = props;
 
   let Component = undefined;
-  if (isMuted) {
-    Component = IconMute;
+
+  if (isMuted && isPrivate) {
+    Component = IconPrivate;
   } else if (isPrivate) {
```

Comment:
You can actually just remove all those changed lines and make `if (isPrivate) {` the start of the `if` block. All that's needed is to remove the special handling of when the stream is muted; let it do the same thing it does as when it's not muted.